next/lib/get-node-page-json-api-params.ts

```diff
@@ -61,8 +61,9 @@ export function getNodePageJsonApiParams(resourceType: ResourceType) {
       "status",
       "metatag",
       "field_excerpt",
+      "path",
     ]);
   }
 
-  return apiParams.getQueryObject();
+  return apiParams;
```

Comment:
**This is a breaking change**

I had to change this so that it returns the apiParams themselves, so I could call the `getQueryObject()` method later on. So I had to change the way this function was called everywhere.